### DISCUSSION TITLE
Map Interface

### DISCUSSION THREAD

**Discussion by iqraaswad** [directus/directus#13600] [Archive]
is it possible to save current position / zoom level when using map interface ? for the example.. when i'm using polygon on specific area.. ex : Dale City.. after marking some place / city / area.. when i click edit / preview on that item... i can't see it, because it was too small.. 

How to prevent map interface using default value / default zoom level ?...

**Comment by benhaynes** [comment]
Yes, administrators can save a default view (zoom and position) globally in the field settings. I don't know if this saves to local storage for individual users though.

![Screen Shot 2022-05-28 at 3 31 00 PM](https://user-images.githubusercontent.com/522079/170840265-08d73f26-dbe1-4be0-9f45-e65614e517c3.png)

**Comment by g05357005** [comment]
It seems the "Default view" only applies when users are NEW a data, 
after user saved point or polygon, then click to edit the entry, 
the opened page is not followed by the default view setting.

if directus want to use openstreetmap for automatically display the geo info, 
maybe following the room level in Default view , or put a setting in project setting?

**Reply by jike212** [reply]
Adding to this, it would be helpful for there to be a default map center and zoom level set for new items (e.g. a full country) but have another center and zoom level for existing entries.

If there's a pin or polygon already saved as part of the item, I don't need to see the whole country, I just need to see that pin or polygon centered in the map.
